app/helpers/exercise_input_helper.rb

```diff
@@ -47,15 +47,34 @@ def should_render_need_help_dropdown?(assignment, organization = Organization.cu
     !assignment.passed? && organization.ask_for_help_enabled?
   end
 
-  def render_submit_button(exercise)
+  def render_submit_button(exercise, assignment=nil)
```

Comment:
In all the two locations that this helper is used, `@assignment is available`, so there it no need to make it optional. Also, it is redudant - see my comment above.

Reply:
Yes, but i thought it didn't make sense to cap kids exam. Know that the capping is optional, we could fix this.

Reply:
Yes, but it is not obvious you have to pass assignment just for caps